manage.py

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# 
-# 
-# 
-# Copyright (c) 2009-2013 University of Dundee. 
```

Comment:
No strong opinion on this. I am fine with keeping the range since this code effectively comes from 2009 even if we didn't keep the history. But 2015 would work as well /cc @joshmoore @jburel